reference/speaker.md

```diff
@@ -27,9 +27,9 @@ void wb_speaker_play_sound(WbDeviceTag left, WbDeviceTag right, const char *soun
 
 **Description**
 
-This function allows the user to play a sound file. Currently only wave files are supported. The function takes as arguments two speaker `WbDeviceTag` respectively for the left and right channels. If both channels should be played on the same speaker or the file has only one channel, it is possible to pass the same device tag for both left and right arguments. Alternatively, if one channel should be ignored, it is possible to pass `0` instead of one of the two tags.
+This function allows the user to play a sound file. All major audio formats are supported. For an exhaustive list, see [here](https://en.wikipedia.org/wiki/FFmpeg#Supported_formats). The function takes as arguments two speaker `WbDeviceTag` respectively for the left and right channels. If both channels should be played on the same speaker or the file has only one channel, it is possible to pass the same device tag for both left and right arguments. Alternatively, if one channel should be ignored, it is possible to pass `0` instead of one of the two tags.
```

Comment:
I would mention at least the most well known audio formats supported in addition to the wikipedia reference, as the wikipedia table is not easy to read and include both video and rare audio formats.